cdap-app-fabric/src/test/java/co/cask/cdap/AppWithSchedule.java

```diff
@@ -46,23 +55,59 @@ public void configure() {
       setDescription("Sample application");
       ObjectStores.createObjectStore(getConfigurer(), "input", String.class);
       ObjectStores.createObjectStore(getConfigurer(), "output", String.class);
-      addWorkflow(new SampleWorkflow());
+      AppConfig config = getConfig();
+      addFlow(new AllProgramsApp.NoOpFlow());
```

Comment:
To keep this app  like before how about adding the flow only if addWorkflow is set to false. If there are unit tests which assert program list  and other stuff those are going to break in your build.